### DISCUSSION TITLE
Добро пожаловать в обсуждения!

### DISCUSSION THREAD

**Discussion by kkohno** [Sokol-Games-GmbH/PixelCombats.PublicApi#17] [General]
## 👋 Добро пожаловать!
  Мы используем Обсуждения как место для общения с другими членами нашего сообщества для следующих целей:
  * Предлагайте улучшения.
  * Задавайте вопросы, которые вас интересуют.
  * Взаимодействуйте с другими участниками сообщества.
  * Приветствуйте других и будьте открытыми. Помните, что это сообщество, которое мы строим вместе 💪.

**Comment by Ygoslavya** [comment]
Круто

**Comment by NezoNa** [comment]
Как возможно попасть в команду разработки?
